src/SIM/SIMoutput.C

```diff
@@ -270,6 +270,9 @@ void SIMoutput::preprocessResPtGroup (std::string& ptFile, ResPointVec& points)
 
 bool SIMoutput::writeGlvG (int& nBlock, const char* inpFile, bool doClear)
 {
+  if (!opt.vtf.empty())
+    inpFile = opt.vtf.c_str();
+
```

Comment:
This can be potentially dangerous.
Because inptFIle=null on input means we shall not open a new file (use in adaptive simulations, e.g.).
Check this.

Reply:
fixed.

Reply:
and changed.